assignments/assignment-8/assignment-8.6.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment-8.6</title>
+    </head>
+    <body>
+        <form>
+            <h2>Event RSVP Application Form:</h2>
+
+            <label for="fullNameInput">Full Name</label>
+            <input
+                type="text"
+                placeholder="Full Name"
+                id="fullNameInput"
+                name="fullName"
+                required
+            />
+            <label for="emailInput">Email</label>
+            <input
+                type="email"
+                placeholder="Email"
+                id="emailInput"
+                name="email"
+            /><br /><br />
+
+            <h3>Will you attend the event</h3>
+            <label for="attendEventInput">Yes</label>
+            <input type="radio" id="yesInput" name="attend" />
+            <label for="attendEventInput">No</label>
+            <input type="radio" id="noInput" name="attend" />
+
+            <h3>If yes, how many guests will you bring?</h3>
+            <label for="quantityInput"
+                >Quanity of guests (between 1 and 20)</label
+            >
+            <input
+                type="number"
+                id="quantityInput"
+                name="guests"
```

Comment:
numberOfGuests would be a good choice for the name attribute.

Reply:
ok